### ISSUE TITLE
Adopt activityAsCase German special to contacts

### ISSUE THREAD

**Issue by JaquM** [hzi-braunschweig/SORMAS-Project#5075]
### Situation Description

The activities of a person in critical situations should also be able to be recorded already in contacts and if the person becomes infected also be transferred to the case.

### Feature Description

The Germany-specific field ActivityasCase should be able to be entered in exactly the same way in the contacts. For this purpose, Issue [#4761](https://github.com/hzi-braunschweig/SORMAS-Project/issues/4761) can also be implemented in the same way in the contact entity. The recorded data should be taken over as with the exposures when converting a contact into a case.

**Comment by MateStrysewske** [comment]
@JaquM I don't agree with this change request. A contact in SORMAS is about a specific contact EVENT, not about the existence of a person as a contact. Such a contact event can not have any further activities beyond the event of exposure. Therefore, this request contradicts the purpose of contacts in the system.

**Comment by JaquM** [comment]
@MateStrysewske 
I agree with you about the international meaning of the field. As the field has a different meaning in Germany, namely the recording of care, accommodation or activity in sensitive facility structures, these are relevant for suspects of infection as well as for cases. We can talk about the transfer from contact to case, the collection of data remains relevant.

**Comment by MateStrysewske** [comment]
@JaquM The technical and consequentially semantic nature of contacts in SORMAS doesn't change depending on the country the system is used in though. I understand that the field itself has a different meaning in Germany, but my main point remains: that contacts in SORMAS are contact events, i.e. individual instances of contacts with (suspected) cases, and NOT a description of the associated person as a contact. For such contact events, it is semantically incorrect to track any data beyond this particular contact event.

If data like this needs to be collected independent of whether or not a person is a case or contact, we might need to move them out of the epidemiological data and attach them to the person itself instead. However, that also means that this data would potentially be disconnected from the entities that it is actually relevant for.

**Comment by JaquM** [comment]
@MateStrysewske 
I realize that, that's exactly the point. What has the person done as a contact in the period in which he is recorded as a contact in the system. So they are staying or working in a hospital, nursing home, etc.? This is data that should not be recorded for the person in general, but for the contact process, as mostly the 14 days of the incubation period.

**Comment by bernardsilenou** [comment]
@JaquM @MateStrysewske @kwa20 
As Maté already explained, the main issues I can see here is mixing person based wiht entity bases system like SORMAS. The entity based system can model much of the complex relationships in real life that a person based can not. In addition if we have the entity based, then we can aggregate to get the person based.
- Adding such data as suggested is fine but would not be best to add to the contact as they are. Maybe to the person? we take this one for an internal refinement.
- Acridity as a case is actually a contact by its own right, having the same person. Activity as a case means at the point in time of the event, the contact person was already a case, thus contradicting the definition of activity as a case and contact.
